### ISSUE TITLE
Cannot preinstall WT package 1.4.3141.0 Windows10_PreinstallKit.zip on Windows Server 10.0.20251.1000

### ISSUE THREAD

**Issue by PGomersall** [microsoft/terminal#8243]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd] Windows Server 10.0.20251.1000
Windows Terminal version (if applicable): N\A

Any other software?
```

# Steps to reproduce

1) Open elevated Powersell
Run Add-AppxProvisionedPackage -Online -FolderPath C:\WT
Add-AppxProvisionedPackage : No applicable main package was found for this platform.
The directory must contain an applicable package with a .main file extension. For more information, see the help.
# Expected behavior

# Actual behavior

Installs WT for all users

**Comment by zadjii-msft** [comment]
Wait _what_? where are you getting this `PreinstallKit.zip` file? That doesn't look like something that _we_ ship. IIRC we're only shipping an `msixbundle` for the packaged [release](https://github.com/microsoft/terminal/releases)

**Comment by WSLUser** [comment]
https://github.com/microsoft/terminal/releases/download/v1.4.3141.0/Microsoft.WindowsTerminal_1.4.3141.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip

**Comment by WSLUser** [comment]
@DHowett started issuing them the last release or two ago.

**Comment by PGomersall** [comment]
Yes as @WSLUser notes: https://github.com/microsoft/terminal/releases/download/v1.4.3141.0/Microsoft.WindowsTerminal_1.4.3141.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip

**Comment by zadjii-msft** [comment]
Alright well, I'll defer to Dustin then because I have no idea what that is 😄

**Comment by DHowett** [comment]
So you did this successfully on Windows Client? Interesting! Can you grab a copy of `C:\Windows\Logs\DISM.log` after a failed apply?

We’ve only validated that the PreinstallKit works on Client, so I’m not surprised that there might be an issue here.

**Comment by PGomersall** [comment]
@DHowett Yes works on most recent W10 client: 10.0.19042.630
Here you go with the log file:
[dism.log](https://github.com/microsoft/terminal/files/5531960/dism.log)

**Comment by DHowett** [comment]
This is really strange! Every failure says:

```
    Msg:[Failed while adding license file '.\6520b63bb369427b9cfbf867f6ab2cb4.msixbundle'] 
```

It's as though it's attempting to parse the bundle itself as a license. That seems deeply incorrect.

I'll reach out to the DISM team and see if I can reproduce this on Server myself. :smile:
